### DISCUSSION TITLE
Nieuw homescreen voor Confac [Work In Progress]

### DISCUSSION THREAD

**Discussion by Thomasjanssens001** [be-pongit/confac-front#238] [Ideas]
### **User story voor nieuw homescreen (nog officieel te noteren):**
- Als een gebruiker wil ik, bij het inloggen in Confac, een overzicht hebben van belangrijke metingen en visualisaties van de metingen om een overzicht te krijgen van de actionable data.
### **Requirements voor nieuw homescreen (nog officieel te noteren)**
- Het nieuwe homescreen moet altijd de eerste landingspagina zijn bij het inloggen in Confac
- Het nieuwe homescreen moet in de vorm van een nieuwe tab toegevoegd worden aan de bestaande tabs met de naam "Home"
- **Metingen onder project** 
             - De hoeveelheid projecten waar alle contracten getekend zijn (zowel in absolute cijfers als in %)
             - De hoeveelheid projecten zonder contract (raamcontract+werkcontract) + kleurcodes (0-15 dagen zonder contracten -> groen; 16-30 dagen -> oranje; 31+ dagen -> rood)
             - Een overzicht van het aantal projecten + een line chart die de evolutie per jaar/maand toont. In de grafiek moet er de mogelijkheid zijn om in of uit te zoomen om meer de details of een overzicht van de data te bekijken
             - De omzet per kalenderjaar van alle lopende projecten
- **Metingen onder klanten** 
             - De hoeveelheid klanten met lopende projecten+ een line chart die de evolutie per jaar/maand toont. In de grafiek moet er de mogelijkheid zijn om in of uit te zoomen om meer de details of een overzicht van de data te bekijken. In de grafiek moeten 2 linecharts zijn, 1 met de evolutie van klanten overheen de tijd en de andere met de evolutie van projecten overheen de tijd.
             - De omzet per kalenderjaar van alle lopende projecten
- **Metingen onder facturen** 
              - De hoeveelheid niet-geconfirmeerde facturen waarbij de eerste verstuurdatum ouder is dan 35 dagen
              - De hoeveelheid facturen met een niet-verstuurde email (zowel in absolute cijfers als in %)
              - De laatst gecontroleerde datum van betaling per factuur
              - Een overzicht met alle openstaande facturen + kleurcodes (0-15 dagen open -> groen; 16-30 dagen -> oranje; 31+ dagen -> rood)
              - Een overzicht van alle facturen zonder een consultant (zowel in absolute cijfers als in %)
              - Een overzicht van alle facturen zonder een projectmaand (zowel in absolute cijfers als in %)
- Het moet mogelijk zijn om in de tabellen te sorteren op basis van de gegeven data (alfabetisch etc.)

**Comment by Laoujin** [comment]
"_Als een gebruiker wil ik, bij het inloggen in Confac, een overzicht hebben van belangrijke metingen en visualisaties van de metingen om een overzicht te krijgen van de actionable data._"

Dit is goed : bij het begin een kort overzicht om duidelijk te maken -- "wat is hier de bedoeling? wat is deze user story" etc. Kort, to the point & duidelijk!


_1) Het nieuwe homescreen moet altijd de eerste landingspagina zijn bij het inloggen in Confac
2) Het nieuwe homescreen moet in de vorm van een nieuwe tab toegevoegd worden aan de bestaande tabs
3 ) Het moet mogelijk zijn om terug te navigeren naar de homepagina na het bezoeken van een andere tab_

Hier zitten we met de proza ;)
1+2) Dit wordt de startpagina van confac. Voeg bovenaan een nieuwe navigate-link toe **met text "NAAM LINK"** (<-- dat is belangrijk, welke tekst)
3) --> Deze is voor mij overbodig. Kan een algemene regel zijn "elke pagina moet bereikbaar zijn". Maar typisch kun je er vanuit gaan dat dit gewoon mogelijk is. Vb door de link uit 2) te gebruiken...


 Voor de rest: a picture is worth a thousand words :)

**Reply by Thomasjanssens001** [reply]
Wireframes volgen nog, ik had vandaag nog op short notice een gesprek voor een project! Vanaf ik wireframes heb zal ik ze hier toevoegen.

Nog een extra vraag, waar gaan we in Git alle user stories en requirements verzamelen? Hier in de discussions?

**Comment by Laoujin** [comment]
Dat laat ik aan jou over...

Als dit jou een goed formaat lijkt, dan wel.
Liefst geen andere website (vb Confluence) want dat is dan verloren tijd / weggesmeten moeite, jij zou de enige zijn die daar ooit naar gaat kijken of het ooit up to date houdt.

Confluence is waarschijnlijk ook markdown...

Enkele mogelijkheden:
- Github Discussions
- Github Issues
- Github Projects
- Word documents
- Markdown
- Any other human readable format

Als je vb kiest voor "Word docs"
--> Dan voegen we folder toe aan confac: "docs" of "analysis"


Voor bugs etc zou ik wel Github Issues blijven gebruiken.

**Comment by Thomasjanssens001** [comment]
wireframe: https://balsamiq.cloud/sslkx8u/p7b7vjl/r6CFF
